### ISSUE TITLE
Windows ver keeps crashing

### ISSUE THREAD

**Issue by Gewgaw** [clementine-player/Clementine#4688]
I have clemintine on windows, mac & linux, but the windows version keeps falling over with the following: 
Problem signature:
  Problem Event Name:	APPCRASH
  Application Name:	clementine.exe
  Application Version:	1.2.0.0
  Application Timestamp:	53578e75
  Fault Module Name:	ntdll.dll
  Fault Module Version:	6.3.9600.17476
  Fault Module Timestamp:	54516af9
  Exception Code:	c0000005
  Exception Offset:	00044990
  OS Version:	6.3.9600.2.0.0.768.100
  Locale ID:	7177
  Additional Information 1:	5861
  Additional Information 2:	5861822e1919d7c014bbb064c64908b2
  Additional Information 3:	a10f
  Additional Information 4:	a10ff7d2bb2516fdc753f9c34fc3b069

Win 8.1, When i run it in my mac or linux vm's on the same machine, no issues.
It doesn't fall over on windows 7.

**Comment by pavelvat** [comment]
>Application Version: 1.2.0.0

1) Last version Clementine is 1.2.3
2) Use debug build of Clementine from http://builds.clementine-player.org/win32/debug/
direct link to last stable version 1.2.3: http://builds.clementine-player.org/win32/debug/ClementineSetup-1.2.3.exe

**Comment by Gewgaw** [comment]
Curious how windows reports 1.2.0 when in fact it is 1.2.3 ?????????????
BUT... i think i found the problem.... GLASSWIRE!!! 
it is a firewall program with a memory leak, causing clementine to crash.
ugh!!

**Comment by ArnaudBienner** [comment]
You mean your OS just ran out of memory and that made Clementine (or probably also other applications) to crash?

**Comment by Gewgaw** [comment]
nope, only clementine, glasswire was taking up 1.3 gig of my 8gb memory, but it had something to do with the firewall's log files getting too big. If I set it to clear daily, the memory usage stays at about 300mb for the f/w and clementine does not crash. Just to test, I set everything back and lo and behold , it would crash , or the playback would become jittery within a day.

**Comment by JulianVolodia** [comment]
Hi @Gewgaw if you still could test new version of Clementine.

**Comment by Gewgaw** [comment]
@JulianVolodia I have abandoned Windows altogether about 3 years ago. I can test on Linux if you would like?

**Comment by JulianVolodia** [comment]
Dear @Gewgaw thanks. If you think there could be something with it, give it a try. Maybe you find something (new).

But afaik from previous comments this issue affects Windows only?

**Comment by JulianVolodia** [comment]
@Gewgaw if you could check if you still see the issue on your side (Linux, or whatever). Problem was reported for 1.2 so could have been patched already.

**Comment by Gewgaw** [comment]
Hi Volodia, 

OK, tried it on DELL Optiplex, HP Elitebook, HP 655, Lenovo E530, Lenovo
G50, Noname desktop, Linux and windows 8.1. Cannot replicate crashing. 

Tried with low memory, 2GB, mid 4GB and 8/16GB. Spinning Rust and SSD.
AMD and NVIDIA and INTEL GFX. None of the configurations had crashes. 

My music library is not large, 6.1 Gb, so it may crash with a larger
music library. 

Cheers!

On 2019-11-15 21:16, Volodia wrote:

> @Gewgaw [1] if you could check if you still see the issue on your side (Linux, or whatever). Problem was reported for 1.2 so could have been patched already. 
> 
> --
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub [2], or unsubscribe [3].
 

Links:
------
[1] https://github.com/Gewgaw
[2]
https://github.com/clementine-player/Clementine/issues/4688?email_source=notifications&amp;email_token=ACPGRWLMDH2UAHVARUX644TQT3YO3A5CNFSM4AZX6KW2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEEGN55Q#issuecomment-554491638
[3]

**Comment by JulianVolodia** [comment]
@Gewgaw thanks it's enought I think. If somebody will see no issue about it and test against new version then we will look again. Thanks for contribution.

If you could please close the issue, because as Author of this you have the power. : - )

**Comment by Gewgaw** [comment]
Good Grief! I thought you wanted me to test on something new!! If you
were referring to the issue five years ago, it was glasswire ( a certian
version of glsswire and a certain version of clementine,) causing it. I
doubt I even have the Github details any more. XD. Good to know it
actually gets read. Thanks for actually investigating it and your time.
I have moved to Linux since then and havent been happier! I will scratch
for the details and blat the error report. If you ever need me to test
something, let me know, I will be happy to help. 

On 2019-11-25 01:10, Volodia wrote:

> @Gewgaw [1] thanks it's enought I think. If somebody will see no issue about it and test against new version then we will look again. Thanks for contribution. 
> 
> If you could please close the issue, because as Author of this you have the power. : - ) 
> 
> --
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub [2], or unsubscribe [3].
 

Links:
------
[1] https://github.com/Gewgaw
[2]
https://github.com/clementine-player/Clementine/issues/4688?email_source=notifications&amp;email_token=ACPGRWIASHR4I5REEI6M4LTQVMCXZA5CNFSM4AZX6KW2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEFAXPZQ#issuecomment-557938662
[3]

**Comment by JulianVolodia** [comment]
> Thanks for actually investigating it and your time. I have moved to Linux since then and havent been happier!

It is all because of your very first comment and your interesting in it.

If it not longer seen on Linux and Windows as you try to test it, please happily close the issue :)

**Comment by JulianVolodia** [comment]
> None of the configurations had crashes.

> Linux and windows 8.1. Cannot replicate crashing.

As I correctly understand - no issues last days seen? If some get out again, we will look at that against new issue report. Now we have to deal with Ancient Rune Issues. ;D

**Comment by JulianVolodia** [comment]
I use Linux and Windows, all of tools have their pros and cons. Have a nice day!
